README.md

```diff
@@ -15,7 +25,7 @@ The Senzing g2-sdk-go-next packages provide a
 language Software Development Kit that wraps the
 Senzing C SDK APIs.
 
-[![License](https://img.shields.io/badge/License-Apache2-brightgreen.svg)](https://github.com/Senzing/g2-sdk-go-next/blob/main/LICENSE)
```

Comment:
@docktermj I assumed this was incorrectly marked as `g2-sdk-go-next` but please correct me if I am wrong about that.

Reply:
@kernelsam Your change is correct. Good catch!